src/tike/ptycho/__init__.py

```diff
@@ -1,39 +1,7 @@
-"""Provides ptychography solvers.
+"""Provide ptychography solvers and tooling.
```

Comment:
What dose tooling mean?

Reply:
Functions needed to develop and use something. i.e. a ptychography simulator, functions which generate initial guesses for the probe, anything that is not a solver, but you might need to use or develop a solver.